packages/gasket-request/lib/wrappers.js

```diff
@@ -0,0 +1,26 @@
+import { GasketRequest } from './request.js';
+
+/** @type {import('./internal.d.ts').withGasketRequestCache} */
```

Comment:
If you make my suggested change of correcting this so it's an export of a function, not a type, you can do this instead:

```suggestion
/** @type {typeof import('./internal.d.ts').withGasketRequestCache} */
```